build.sc

```diff
@@ -75,6 +75,7 @@ object Deps {
   val zinc = ivy"org.scala-sbt::zinc:1.4.0-M1"
   val bsp = ivy"ch.epfl.scala:bsp4j:2.0.0-M4"
   val jarjarabrams = ivy"com.eed3si9n.jarjarabrams::jarjar-abrams-core:0.3.0"
+  val proguard = ivy"com.guardsquare:proguard-core:7.0.0"
```

Comment:
Why is this dependency needed. Don't we resolve it in the `proguardClasspath` target at runtime?

Reply:
Whoops, this was from when I was starting and didn't realize what I was doing. I forgot to remove it